src/js/themes/base.d.ts

```diff
@@ -982,6 +985,71 @@ export interface ThemeType {
         color?: ColorType;
       };
     };
+    checkBoxGroup?: {
```

Comment:
can we pull out the inputs then do something like 

```
 [(or just check input list): string]: {
    container?: {
      extend?: ExtendType;
    };
  };
```

I know we do something similar with `[size]` and other areas

Reply:
I originally tried this approach but was getting type errors on the other `theme.formfield.` objects because everything was trying to match the 
```
 container?: {
      extend?: ExtendType;
    };
```
type